pkg/controller/controller_test.go

```diff
@@ -291,7 +291,7 @@ func TestValidateForm(t *testing.T) {
 		},
 	})
 
-	controller := New(nil, p, []string{})
+	controller, _ := New(nil, p, []string{}, context.Background())
```

Comment:
Don't ignore error

Reply:
Updated in the latest version as you suggested in another comment.